src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -194,7 +194,7 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin, EntityFilter
 
 .people-roles-summary-header {
   display: flex;
-  background-color: $gray1;
+  background-color: rgba(1, 51, 102, 0.15);
```

Comment:
oops, this was my request but come to think of we should change this to colour to one in our base theme that is as close to this colour as possible.  We might have to change this in the `SummaryDefineCompany` line 98 as well.  If i had to guess.... the theme colour closest to this might be `$BCgovBlue8`.

Reply:
I tried $BCgovBlue8. The text is not visible. So will unbranch the file from this PR. looks like more styling is required

Reply:
This is fine for now! I would leave it in so it at the very least matches the `summarydefinecompany`. We can define and update to a theme colour later 👍